packages/flutter/lib/src/material/flexible_space_bar.dart

```diff
@@ -137,6 +138,16 @@ class FlexibleSpaceBar extends StatefulWidget {
   /// padding of 72.0 pixels and bottom padding of 16.0 pixels.
   final EdgeInsetsGeometry? titlePadding;
 
+  /// Defines the padding applied to the [title]
+  /// when the [FlexibleSpaceBar] is collapsed.
+  ///
+  ///This allows for separate control of the title's padding
```

Comment:
missing space after /// here and below

Reply:
Hi @cedvdb glad you found time to review the changes. What do you mean with missing space? Do you mean the blank lines on line 143 and 146? I simply follow the [code guide](https://github.com/flutter/flutter/blob/master/docs/contributing/Style-guide-for-Flutter-repo.md#leave-breadcrumbs-in-the-comments) and the description of other attributes like 'titlePadding'.

Reply:
Oh i am so sorry :D. I know what you mean. I will fix that.